proxy/api/src/http/error.rs

```diff
@@ -105,6 +105,11 @@ pub async fn recover(err: Rejection) -> Result<impl Reply, Infallible> {
                         "GIT_ERROR",
                         format!("Internal Git error: {:?}", git_error),
                     ),
+                    coco::Error::NoBranches => (
+                        StatusCode::BAD_REQUEST,
+                        "GIT_ERROR",
+                        "The repository has no branches".to_string(),
```

Comment:
To keep the error messages consistent we should do the following:
```suggestion
                        coco::Error::NoBranches.to_string(),
```